cmd/docker-app/root.go

```diff
@@ -21,8 +23,10 @@ func newRootCmd(dockerCli command.Cli) *cobra.Command {
 			}
 			return nil
 		},
+		Version: fmt.Sprintf("%s, build %s", internal.Version, internal.GitCommit),
```

Comment:
Wasn't sure if `build <sha>` or `commit <sha>` was more desirable; also currently, the version itself contains a short-commit sha, but possibly that isn't the case on tagged releases